pkg/ldap/client_test.go

```diff
@@ -1,10 +1,10 @@
 // +build integration
-
+// Note the comment above MUST BE ON THE FIRST LINE.
```

Comment:
1 blank space between tag and next comment.

Reply:
lovely